### ISSUE TITLE
Question: electrum patch

### ISSUE THREAD

**Issue by rkovacevic** [bitkey/bitkey#20]
I assume this is a debian patch, but is there any documentation somewhere about what it does, and why it is needed?

**Comment by JedMeister** [comment]
It just tweaks electrum. You can view exactly what it does here on GitHub: https://github.com/heneault/bitkey/blob/master/overlay/usr/local/src/electrum.patch

The lines in red are removed, the lines in green are added.

**Comment by rkovacevic** [comment]
Right, but I was hoping for a quick explanation of what it does, so I don't have to go dig through electrum source.

**Comment by JedMeister** [comment]
Apologies if my answer wasn't very helpful.

Alon & Liraz did the initial BitKey development and I wasn't involved with that. I have only supervised and provided feedback on the most recent maintenance update (plus some testing). Also please note that Alon (or Liraz) will be doing a final code review prior to release (that's why we haven't yet built and released it - they're both tied up with other projects).

My understanding (and reading of the patch) is that this it is to tweak electrum to suit the BitKey use case. I'm not a regular bitcoin user and I haven't read through the electrum source either. The purpose of my initial code review was purely to make sure nothing nasty was creeping in and to make sure that it basically worked. 

I'll ask whoever does the final code review to give a quick overview of the patch. Not sure when that will be. I hope soon because I'd love to see it released ASAP.

**Comment by rkovacevic** [comment]
Excellent, I think it would be great to document it, so that casual users can easily audit. Thx.

**Comment by lirazsiri** [comment]
Jeremy has it about right. The patch modifies Electrum just a bit so that the usage scenario we're trying to optimize for has less room for error. Otherwise if you choose the wrong option, things could break.
